src/SearchResults.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+
+const headings = [
+  `id`,
+  `title`,
+  `firstName`,
+  `surname`,
+  `email`,
+  `roomId`,
+  `checkInDate`,
+  `checkOutDate`,
+  `numberOfNights`,
+  `profile`
+];
+
+function SearchResults(props) {
+  const [id, setId] = useState(null);
+
+  function handleButtonClick(clickedId) {
```

Comment:
I don't think you need this function as you should be able to handle `setId` function as the prop, i.e.
```
<BookingItem
                booking={booking}
                key={index}
                handleClick={setId}
              />
```